templates/www/us/2019/devconfus-2019-eventpage.html

```diff
@@ -73,9 +73,10 @@ <h3>{{ ('Important dates') }}:</h3>
             <li>{{ _('CfP Opens') }}: <b>{{ _('January 27th, 2019') }}</b></li>
             <li>{{ _('CfP Closes') }}: <b>{{ _('April 8, 2019') }}</b></li>
             <li><strike>{{ _('Accepted speakers confirmation') }}: <b>{{ _('May 16th, 2019') }}</b></strike></li>
-            <li>{{ _('Accepted speakers confirmation') }}: <b>{{ _('May 27th, 2019') }}</b></li>
-            <li>{{ _('Schedule published') }}: <b>{{ _('June 18th, 2019') }}</b></li>
-            <li>{{ _('Registration opens') }}: <b>{{ _('June 18th, 2019') }}</b></li>
+            <li><strike>{{ _('Accepted speakers confirmation') }}: <b>{{ _('May 27th, 2019') }}</b></strike></li>
```

Comment:
Typo?

Reply:
no, striking out the old date and putting in a new one. Unless I am not seeing the typo.